src/twisted/python/test/_deprecatetests.py.3only

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
```

Comment:
Right now, these arcane contortions are needed to make forward progress, and there is no other clean way.  As time goes on, I would like to see twistedchecker and pydoctor work on Py3, so that we can run them on Py3 as part of the automation on Travis.
That way, people can just focus on writing code, and not have to do
contortions to appease broken tools.

I've submitted a few patches to both those tools to get the ball rolling:

https://github.com/twisted/pydoctor/pulls/rodrigc
https://github.com/twisted/twistedchecker/pulls/rodrigc